### PR TITLE
Update telemetry_fields.yaml

### PR DESCRIPTION
**Pull request by Karthi8883**
added telemetry field for UPS Bypass reversed status

### CODE REVIEW

**Comment by Karthi8883** [comment]
we have raised this  request to test the access to master.

File: ontology/yaml/resources/ELECTRICAL/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -49,4 +49,5 @@ IOBM:
   - output_phase3_current_sensor
   - bypass_phase1_neutral_voltage_sensor
   - bypass_phase2_neutral_voltage_sensor
-  - bypass_phase3_neutral_voltage_sensor
\ No newline at end of file
+  - bypass_phase3_neutral_voltage_sensor
+  - bypass_phase_reversed
```
[the remark this answers is not in the document]
Comment (Karthi8883) [reply ontology/yaml/resources/ELECTRICAL/entity_types/ABSTRACT.yaml]: This binary point provides the status of bypass phase, whether it is normal or reversed. modified the subfield to status now. hope this is fine.

**Comment by mschulze17** [comment]
@tasodorff back to you

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 2]
```diff
@@ -4946,3 +4946,11 @@ EFHLC:
   - high_exhaust_fan_speed_status
   implements:
   - OPERATIONAL
+
+PM:
+  description: "Particle Concentration monitoring."
+  is_abstract: true
+  uses:
+  - zone_particle_concentration_sensor
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:4954]: air or water? you can use `particle_concentration_sensor` which already exists, or you can add one for `zone_air_particle_concentration_sensor`.

**Review by tasodorff — CHANGES_REQUESTED** [review]
see comments

File: ontology/yaml/resources/ELECTRICAL/entity_types/ABSTRACT.yaml [hunk 3]
```diff
@@ -49,4 +49,5 @@ IOBM:
   - output_phase3_current_sensor
   - bypass_phase1_neutral_voltage_sensor
   - bypass_phase2_neutral_voltage_sensor
-  - bypass_phase3_neutral_voltage_sensor
\ No newline at end of file
+  - bypass_phase3_neutral_voltage_sensor
+  - bypass_phase_status
```
[the remark this answers is not in the document]
Comment (Karthi8883) [reply ontology/yaml/resources/ELECTRICAL/entity_types/ABSTRACT.yaml:53]: Updated in opt_uses. Kindly verify.

**Comment by tasodorff** [comment]
resolve conflicts so we can run validation. ensure validation passes for every PR before submitting. you can use local validator tools to do this.

**Comment by Karthi8883** [comment]
Updated as part of [https://github.com/google/digitalbuildings/pull/669/]

**Pull request closed without merging**